x-pack/packetbeat/magefile.go

```diff
@@ -32,6 +32,11 @@ func init() {
 
 	devtools.BeatDescription = "Packetbeat analyzes network traffic and sends the data to Elasticsearch."
 	devtools.BeatLicense = "Elastic License"
+	if devtools.Platform.GOOS == "windows" {
```

Comment:
While debugging the license checking issues I found that this is not run during packaging. This means that this hold won't constrain the license note customisation though it will prevent escape of the actual installer. Is this a problem and if it is where would be the best equivalent place in the packaging code path?

Reply:
I think this block is not necessary. All artifacts produced by running `mage package` from beats/x-pack/packetbeat directory are ELv1 (and all artifacts from beats/packetbeat are ASL2).

Reply:
OK. This was a defensive inclusion in case beats/x-pack/packetbeat was moved to ELv2 which was something that was raised as a concern.